# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParent.h
# Commit: 14998637bb01
# Full Hash: 14998637bb011b58e048aec43461ad6e80baeadc
# Author: Randell Jesup <rjesup@wgate.com>
# Date: 2022-09-10 21:45:26
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Simplify FileSystemManagerChild by using the VirtualSendImpl attribute in PFileSystemManager definition; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155352
# ==============================================================================

diff -r aa682026c2d5 -r 14998637bb01 dom/fs/parent/FileSystemManagerParent.h
--- a/dom/fs/parent/FileSystemManagerParent.h	Sat Sep 10 02:15:02 2022 +0000
+++ b/dom/fs/parent/FileSystemManagerParent.h	Sat Sep 10 02:15:03 2022 +0000
@@ -36,30 +36,26 @@
 
   void AssertIsOnIOTarget() const;
 
-  mozilla::ipc::IPCResult RecvGetRootHandleMsg(
-      GetRootHandleMsgResolver&& aResolver);
+  mozilla::ipc::IPCResult RecvGetRootHandle(GetRootHandleResolver&& aResolver);
 
-  mozilla::ipc::IPCResult RecvGetDirectoryHandleMsg(
+  mozilla::ipc::IPCResult RecvGetDirectoryHandle(
       FileSystemGetHandleRequest&& aRequest,
-      GetDirectoryHandleMsgResolver&& aResolver);
+      GetDirectoryHandleResolver&& aResolver);
 
-  mozilla::ipc::IPCResult RecvGetFileHandleMsg(
-      FileSystemGetHandleRequest&& aRequest,
-      GetFileHandleMsgResolver&& aResolver);
+  mozilla::ipc::IPCResult RecvGetFileHandle(
+      FileSystemGetHandleRequest&& aRequest, GetFileHandleResolver&& aResolver);
 
-  mozilla::ipc::IPCResult RecvGetFileMsg(FileSystemGetFileRequest&& aRequest,
-                                         GetFileMsgResolver&& aResolver);
+  mozilla::ipc::IPCResult RecvGetFile(FileSystemGetFileRequest&& aRequest,
+                                      GetFileResolver&& aResolver);
 
-  mozilla::ipc::IPCResult RecvResolveMsg(FileSystemResolveRequest&& aRequest,
-                                         ResolveMsgResolver&& aResolver);
+  mozilla::ipc::IPCResult RecvResolve(FileSystemResolveRequest&& aRequest,
+                                      ResolveResolver&& aResolver);
 
-  mozilla::ipc::IPCResult RecvGetEntriesMsg(
-      FileSystemGetEntriesRequest&& aRequest,
-      GetEntriesMsgResolver&& aResolver);
+  mozilla::ipc::IPCResult RecvGetEntries(FileSystemGetEntriesRequest&& aRequest,
+                                         GetEntriesResolver&& aResolver);
 
-  mozilla::ipc::IPCResult RecvRemoveEntryMsg(
-      FileSystemRemoveEntryRequest&& aRequest,
-      RemoveEntryMsgResolver&& aResolver);
+  mozilla::ipc::IPCResult RecvRemoveEntry(
+      FileSystemRemoveEntryRequest&& aRequest, RemoveEntryResolver&& aResolver);
 
   mozilla::ipc::IPCResult RecvCloseFile(FileSystemGetFileRequest&& aRequest);
 